### PR TITLE
Enhancement: ApiKey Creation from Admin

### PR DESCRIPTION
**Pull request by aybruhm**
The following was done:

- removed expiry_time field and altered scope field on client apikey model
- override save model method on client apikey admin

### CODE REVIEW

File: Univast/throttling.py [hunk 1]
```diff
@@ -37,15 +51,14 @@ def get_cache_key(self, request: HttpRequest, view):
...
-        self.get_rate()
+        self.get_rate(request)
 
         return self.cache_format % {
             "scope": client_scope,
             "ident": api_key,
         }
 
-    def get_rate(self):
-        request = self.request
```
Comment (aybruhm) [inline Univast/throttling.py:48]: When you refactored this, did you test it?

Comment (devgenix) [reply Univast/throttling.py:48]: I haven't run tests yet, but I tested the API and accessing pages and they worked fine. 

I'm still on it.

Comment (aybruhm) [reply Univast/throttling.py:48]: Alright.

Comment (devgenix) [reply Univast/throttling.py:48]: > Alright.

Tests passed, all good.

**Comment by devgenix** [comment]
@aybruhm 

In addition, the following has officially been done:

- [x] Set cache to use Redis. 
- [x] Added webhook to notify API Clients of changes in the database.

**Comment by aybruhm** [comment]
> @aybruhm 
> 
> In addition, the following has officially been done:
> 
> - [x] Set cache to use Redis. 
> - [x] Added webhook to notify API Clients of changes in the database.

Awesome work, man!

**Review by devgenix — COMMENTED** [review]
Beautiful, beautiful. I'm feel happy with these new updates.

**Pull request merged**